lang/src/tests/mod.rs

```diff
@@ -240,6 +299,7 @@ fn multiple_states() {
 fn multiple_deploy_handlers() {
     assert_failure(
         quote! {
+            #![env = ink_core::env::DefaultSrmlTypes]
```

Comment:
Would it maybe be a good idea to make `ink_core::env::DefaultSrmlTypes` be a default to not break too many contracts? Maybe this is a bad idea. Let's discuss this in an issue.

Reply:
https://github.com/paritytech/ink/issues/127